Configuration/PyReleaseValidation/test/test_mc_setup/test_MC_setup.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# This script mimics a MC chain with HLT run 
+# a in a different release from the current one
+
+echo '#### MC Setup'
+
+# this would make it run only in 14_0_X releases, need to keep?
+
+# if [[ ! ("$CMSSW_VERSION" == *"14_0_"*) ]]; then
+#     echo "This is not a 14_0_X release. No need to test this. Exiting."
+#     exit 0
+# fi
+
+# inputs
+conditions=$1 
+era=$2
+hlt=$3
+release_hlt=$4
+gt_hlt=$5
+beamspot=$6
+
+base_dir=$PWD
+base_cms=$CMSSW_BASE
+base_arch=$SCRAM_ARCH
+
+
+# to take into account changes in gcc
+base_arch_no_gcc=$(echo $SCRAM_ARCH | cut -d "_" -f -2)
+if [[ ! ("$CMSSW_VERSION" == $release_hlt) ]]; then
+    hlt_arch=$(echo $(ls -dtrh /cvmfs/cms.cern.ch/${base_arch_no_gcc}*/cms/cmssw/${release_hlt} | tail -1) | cut -d "/" -f 4)
```

Comment:
@smuzaffar Is there a better way to figure out the installation area of "the other release" than hardcoding a dependence on CVMFS?

Reply:
perhaps using $CMS_PATH [indeed, building in cvmfs assumptions is best avoided]

Reply:
one can use scram itself to find the cmssw release e.g.
```
scram list -c $CMSSW_VERSION | grep $CMSSW_VERSION | sed 's|.* ||'
```

note that this test should be run conditionally as not all hlt releases will be available for all `base_arch_no_gcc`

Reply:
> `scram list -c $CMSSW_VERSION | grep $CMSSW_VERSION | sed 's|.* ||'`

Isn't this assuming I know the full `SCRAM_ARCH` for the target HLT release? Here I was trying to infer the newest available gcc for the target release. I agree I would need to add a check that such a release is actually available.